tests/unit/CreateSpell.spec.js

```diff
@@ -0,0 +1,35 @@
+import CreateSpell from '@/components/createspell/CreateSpell.vue'
```

Comment:
I found this [Medium article](https://medium.com/@sarngru/vue-router-unit-testing-navigation-6cc0b0f86811) on router testing, might come in handy later. might not.